### PR TITLE
Fix intermittent ItWlsSamples, ItElasticLogging and ItElasticLoggingFluentd test failure in olcne pipeline

### PR DESCRIPTION
**Pull request by xiancao**
This is to fix intermittent ItWlsSamples, ItElasticLogging and ItElasticLoggingFluentd test failure in olcne pipeline.

In olcne pipeline, the ItWlsSamples test failed intermittently. It complains the managed server2 failed to start due to domain secret mismatch. 
From the doc, https://oracle.github.io/weblogic-kubernetes-operator/faq/domain-secret-mismatch/ , the possible reason is that the managed-server-2 used a different container image.
Also ItElasticLogging and ItElasticLoggingFluentd failed due to elasticsearch already exists in the default namespace. We need to make sure the elasticsearch uninstalled in the teardown().

### CODE REVIEW

**Review by hzhao-github — APPROVED** [review]

**Comment by hzhao-github** [comment]
LGTM

**Comment by hzhao-github** [comment]
You may wanted to add Jenkins run result

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java [hunk 1]
```diff
@@ -158,7 +161,7 @@ void testSampleDomainInImage(String model) {
     // update domainHomeImageBase with right values in create-domain-inputs.yaml
     assertDoesNotThrow(() -> {
       replaceStringInFile(Paths.get(sampleBase.toString(), "create-domain-inputs.yaml").toString(),
-              "domainHomeImageBase: container-registry.oracle.com/middleware/weblogic:" + diiImageTag,
+              "domainHomeImageBase: container-registry.oracle.com/middleware/weblogic:12.2.1.4",
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java:164]: No hardcoded WLS version on the test class

**Review by anpanigr — APPROVED** [review]
lgtm

**Pull request merged**